### ISSUE TITLE
firebase/testing can't connect to firestore emulator after 0.16.1 version. 

### ISSUE THREAD

**Issue by diginikkari** [firebase/firebase-js-sdk#2417]
### [REQUIRED] Describe your environment

  * Operating System version: MACOS 10.15.1
  * Browser version: not affecting
  * Firebase SDK version: 7.5.2
  * Firebase Product: testing

### [REQUIRED] Describe the problem
connection to emulator firestore instance timeouts after upgrading testing library from 0.16.0 to 0.16.1 or 0.16.2. 

#### Steps to reproduce:

Below code timeouts after upgrading from `"@firebase/testing": "0.16.0"` to `"@firebase/testing": "0.16.1"`

Start firestore emulator `firebase emulators:start --only firestore`

#### Relevant Code:
```
import * as firebase from '@firebase/testing';

const app = firebase.initializeAdminApp({ projectId: 'testing' });
const db = app.firestore();
db.collection('test')
  .doc('doc1')
  .set({ ok: true })
  .then(ref => console.log('set succeeded'))
  .catch(err => console.error('set errored', err));
```

**Comment by rommelpe** [comment]
Thank you for taking time filling this issue. I was able to get the same behavior (https://github.com/rommelpe/2417), and it seems that it persists in v0.16.2. I created an internally report (b/145917871) to see what's causing the issue. I'll let you know once I have an update.

**Comment by yuchenshi** [comment]
This is now fixed on `master` and will be included in the next release this week.

**Comment by Elyx0** [comment]
Rolled back to `16.0` in the meantime as well

**Comment by yuchenshi** [comment]
This is now fixed in 0.16.3.
